plugins/LilyPad/XInputEnum.cpp

```diff
@@ -113,31 +114,31 @@ class XInputDevice : public Device
 
     wchar_t *GetPhysicalControlName(PhysicalControl *c)
     {
-        const static wchar_t *names[] = {
-            L"D-pad Up",
-            L"D-pad Down",
-            L"D-pad Left",
-            L"D-pad Right",
-            L"Start",
-            L"Back",
-            L"Left Thumb",
-            L"Right Thumb",
-            L"Left Shoulder",
-            L"Right Shoulder",
-            L"Guide",
-            L"A",
-            L"B",
-            L"X",
-            L"Y",
-            L"Left Trigger",
-            L"Right Trigger",
-            L"Left Thumb X",
-            L"Left Thumb Y",
-            L"Right Thumb X",
-            L"Right Thumb Y",
-        };
         unsigned int i = (unsigned int)(c - physicalControls);
         if (i < 21) {
+            const static wchar_t *names[] = {
+                L"D-pad Up",
+                L"D-pad Down",
+                L"D-pad Left",
+                L"D-pad Right",
+                L"Start",
+                L"Back",
+                L"Left Thumb",
+                L"Right Thumb",
+                L"Left Shoulder",
+                L"Right Shoulder",
+                L"Guide",
+                L"A",
+                L"B",
+                L"X",
+                L"Y",
+                L"Left Trigger",
+                L"Right Trigger",
+                L"Left Thumb X",
+                L"Left Thumb Y",
+                L"Right Thumb X",
+                L"Right Thumb Y",
+            };
```

Comment:
Still don't know why this was moved. IMO it would actually be better left where it originally was and `i < 21` changed into `i < (sizeof(names) / sizeof(names[0]))` or similar (I didn't check if it works).